MIGRATE.md

```diff
@@ -153,7 +153,7 @@ module.exports = {
 
 In summary, we can see the follow changes were made
 1.  The webpack schema for using loaders has changed
-    - `loaders` is now `module.rules`
+    - `module.loaders` is now `module.rules`
```

Comment:
In webpack v1 this was loaders, please remove this

Reply:
Cleaned the commit containing that change.